src/App.js

```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
+import react from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
```

Comment:
- [ ] As required for the project, kindly make sure you delete all the boilerplate from CRA (text, images, styles,...). You should also delete the reportWebVitals.js, setupTests.js,... Make sure you delete all the non related files of your project.